spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+RSpec.describe UsersController do
+  describe 'GET index' do
+    it 'returns all the users'
+  end
+
+  describe 'GET show' do
+    it 'shows a specific user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.